data/db_demo.yaml

```diff
@@ -12,6 +12,23 @@ user:
     roles:
       - View only
 
+streams:
+  - name: ZTF Public
+    altdata:
+      collection: ZTF_alerts
+      selector: [1]
```

Comment:
Should this be a scalar?

Reply:
Again, it's meant to be used with a `$in` operator. I can add an extra type check, but since it anyways lives in `altdata`, would prefer not to bother :)